### PR TITLE
donation page added and remote events page removed

### PR DESCRIPTION
**Pull request by Digi-D**

### CODE REVIEW

File: web-portal/pages/donate.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    methods: {
+      FillInTheBlank: function (status) {
+        if (status === 'ON') {
+          this.show_blank = false
+        } else {
+          this.show_blank = true
+        }
+      }
+    }
```
Comment (MatthewGidcomb) [inline web-portal/pages/donate.vue]: This data function and method aren't needed and can be removed.

In fact, the entire `script` block could be removed, but we include the empty block in some of the other static pages, so I guess it makes sense to keep it for consistency.

File: web-portal/pages/donate.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+</script>
+
+<style scoped>
+  #fill-in-the-blank{
+    cursor: pointer;
+  }
+</style>
```
Comment (MatthewGidcomb) [inline web-portal/pages/donate.vue]: This can be removed too.

**Review by MatthewGidcomb — APPROVED** [review]

File: web-portal/README.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ This application communicates with an [API server](https://github.com/infinite-i
 
 2. Start the Nuxt production server
  ```bash
- npm run start
+npm run start:dev
```
Comment (chriswininger) [inline web-portal/README.md:67]: 👍

File: web-portal/pages/donate.vue (hunk 2 above)
Comment (chriswininger) [reply web-portal/pages/donate.vue]: nice catch, always good to trim the fat

**Review by chriswininger — APPROVED** [review]

**Pull request merged**